.github/workflows/test.yml

```diff
@@ -118,134 +118,6 @@ jobs:
       - run: pnpm run lint && pnpm run check-engines-override
         working-directory: ./
 
-  #
-  # CLIENT test:functional relationMode
-  #
-  client-relationMode:
```

Comment:
I just moved them below.
Starting the `client` tests first is more useful than starting the `client-relationMode` first